### ISSUE TITLE
boxes.el history list nil

### ISSUE THREAD

**Issue by carnil** [ascii-boxes/boxes#17]
Hi

In Debian I got a report about the boxes.el (I'm not an emacs user myself), so I'm just forwarding it

```
From: Kevin Ryde <user42_kevin@yahoo.com.au>
To: Debian Bug Tracking System <maintonly@bugs.debian.org>
Subject: boxes: boxes.el history list nil
Date: Sun, 06 Dec 2015 16:33:09 +1100

Package: boxes
Version: 1.1.2-3
Severity: minor
Tags: patch
File: /usr/share/emacs/site-lisp/boxes.el

In boxes.el the boxes-history variable is set to (list nil) but nil is
not a string.  You can see the effect by trying to step back to it,

    M-x boxes-command-on-region
    M-p
    =>
    previous-history-element: Wrong type argument: char-or-string-p, nil

Usually an empty history variable is made just by

    (defvar boxes-history nil)
```

Original report is at: https://bugs.debian.org/807153

Regards,
Salvatore

**Comment by tsjensen** [comment]
Thank you Salvatore. @zerotao, I know this is many years old, but can you have a look at it?

**Comment by zerotao** [comment]
Sure, I'll take a look and fix it.

**Comment by tsjensen** [comment]
Fixed in #18 by @zerotao - Thanks!!
